src/translations/en.json

```
@@ -3307,6 +3308,41 @@
             "in_progress": "{device} healing is in progress.",
             "network_heal_in_progress": "A Z-Wave network heal is already in progress. Please wait for it to finish before healing an individual device."
           },
+          "update_firmware": {
+            "title": "Update Device Firmware",
+            "warning": "WARNING: Firmware updates can brick your device if you do not know what you are doing and do not correctly follow the manufacturer's guidance. The Home Assistant and Z-Wave JS team will not be able to help you if you brick your device. Would you still like to continue?",
```

I wonder if we need some more legal lingo here? Something about that we take no responsibility for any damages to your device.
```suggestion
            "warning": "WARNING: Firmware updates can brick your device if you do not correctly follow the manufacturer's guidance. The Home Assistant and Z-Wave JS team will not be able to help you if you brick your device. Would you still like to continue?",
```

---

@blhoward2 may have an idea, but I'm not sure when he's available again.

---

```suggestion
            "warning": "WARNING: Firmware updates can brick your device if you do not correctly follow the manufacturer's guidance. The Home Assistant and Z-Wave JS teams do not take any responsibility for any damages to your device as a result of the firmware update and will not be able to help you if you brick your device. Would you still like to continue?",
```